src/dreamchecker/README.md

```diff
@@ -52,9 +52,11 @@ be enabled:
 #ifdef SPACEMAN_DMM
 	#define RETURN_TYPE(X) set SpacemanDMM_return_type = X
 	#define SHOULD_CALL_PARENT(X) set SpacemanDMM_should_call_parent = X
+  #define FINAL_VAR(X) var/final/X
 #else
 	#define RETURN_TYPE(X)
 	#define SHOULD_CALL_PARENT(X)
+  #define FINAL_VAR(X) var/X
```

Comment:
Do these macros work as written with types, e.g. `FINAL_VAR(datum/reagents/reagents)`?

Maybe something like `#define FINAL_VAR var/final` or `var` then you can write `FINAL_VAR/whatever` would work better. I want to get the ergonomics on this right as far as both typed and untyped vars go.

Reply:
They get detected the same way global/static/tmp etc get detected by the parser, so yes

Reply:
`FINAL_VAR/whatever` might work better yeah